SourceX/controls/devices/game_controller.h

```diff
@@ -8,7 +7,7 @@
 #include "controls/controller.h"
 #include "controls/controller_buttons.h"
 
-#ifndef USE_SDL1
+#if !defined(USE_SDL1) || defined(__SWITCH__)

```

Comment:
This was added because display.cpp needs GameController definition for Switch:

https://github.com/diasurgical/devilutionX/blob/master/SourceX/display.cpp#L147

Reply:
Switch is SDL2 so is it actually needed?

Reply:
It doesn't make sense for this to be needed.

@yuripourre You probably tried building with both SWITCH and USE_SDL1=ON, which is not supported

Reply:
```suggestion
 #ifndef USE_SDL1
```